diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/smoke/DiktatSmokeTest.kt

```diff
@@ -84,10 +88,35 @@ class DiktatSmokeTest : FixTestBase("test/smoke/src/main/kotlin",
     @Tag("DiktatRuleSetProvider")
     fun `regression - should not fail if package is not set`() {
         overrideRulesConfig(listOf(Warnings.PACKAGE_NAME_MISSING, Warnings.PACKAGE_NAME_INCORRECT_PATH,
-            Warnings.PACKAGE_NAME_INCORRECT_PREFIX))
+            Warnings.PACKAGE_NAME_INCORRECT_PREFIX), null)
         fixAndCompare("DefaultPackageExpected.kt", "DefaultPackageTest.kt")
     }
 
+    @Test
+    @Tag("DiktatRuleSetProvider")
+    fun `smoke test #5`() {
+        overrideRulesConfig(emptyList(),
+                mapOf(
+                        Pair(Warnings.HEADER_MISSING_OR_WRONG_COPYRIGHT.name, mapOf(
```

Comment:
Why not use `to` instead of explicitly calling `Pair` constructor? I think it's convention for maps creation.